src/Serialization/upgrades/0.11.3.jl

```diff
@@ -0,0 +1,118 @@
+################################################################################
+# Upgrade Summary
+# When `is_basic_serialization_type(T) == true` the type `Vector{T}` the entries
+# are no longer serialized as dicts, The type is encoded on the Vector at the same level as
+# the entry data. The entries no longer being dictionaries prevents the use of backrefs.
+
+
+push!(upgrade_scripts, UpgradeScript(
+    v"0.11.3", # version this script upgrades to
+    function upgrade_0_11_3(s::DeserializerState, dict::Dict)
+        # file comes from polymake
+        haskey(dict, :_ns) && haskey(dict[:_ns], :polymake) && return dict
+
+        # moves down tree to point where type exists in dict
+        # since we are only doing updates based on certain types
+        # no :type key implies the dict is data
+        if !haskey(dict, :type)
+            upgraded_dict = Dict{Symbol, Any}()
+            for (key, value) in dict
+                if value isa String
+                    upgraded_dict[key] = value
+                elseif value isa Dict{Symbol, Any}
+                    upgraded_dict[key] = upgrade_0_11_3(s, value)
+                else
+                    values = []
+                    for v in value
+                        if v isa String
+                            push!(values, v)
+                        else
+                            push!(values, upgrade_0_11_3(s, v))
+                        end
+                    end
+                    upgraded_dict[key] = values
+                end
+            end
+            return upgraded_dict
+        end
+
+        if dict[:type] == string(backref_sym)
+            backref = s.objs[UUID(dict[:id])]
+
+            # data no longer uses backref and is part of the update
+            backref isa String && return backref
```

Comment:
```suggestion
            backrefed_object = s.objs[UUID(dict[:id])]

            # if the backref points to a string, just use that string
            # instead of the backref
            backrefed_object isa String && return backrefed_object
```